woof-code/rootfs-skeleton/root/.config/gtk-3.0/settings.ini

```diff
@@ -0,0 +1,6 @@
+[Settings]
```

Comment:
Add a copy of this file at /etc/gtk-3.0/settings.ini - applications that run as spot don't obey a configuration file in root's home directory

Reply:
done